apps/src/lib/client/tx.rs

```diff
@@ -57,8 +54,13 @@ const TX_UNBOND_WASM: &str = "tx_unbond.wasm";
 const TX_WITHDRAW_WASM: &str = "tx_withdraw.wasm";
 const VP_NFT: &str = "vp_nft.wasm";
 
-const ENV_VAR_ANOMA_TENDERMINT_WEBSOCKET_TIMEOUT: &str =
-    "ANOMA_TENDERMINT_WEBSOCKET_TIMEOUT";
+/// Timeout for jsonrpc requests to the `/events` endpoint in Tendermint.
+const ENV_VAR_ANOMA_TENDERMINT_RPC_TIMEOUT: &str =
+    "ANOMA_TENDERMINT_RPC_TIMEOUT";
+
+/// Default timeout in seconds for jsonrpc requests to the `/events` endpoint in
+/// Tendermint.
+const DEFAULT_ANOMA_TENDERMINT_RPC_TIMEOUT: u64 = 30;
```

Comment:
We could include events in the name since this is just for the events endpoint e.g. `ENV_VAR_ANOMA_TENDERMINT_EVENTS_RPC_TIMEOUT` or `ENV_VAR_ANOMA_TENDERMINT_EVENTS_MAX_WAIT_TIME`